tests/python/frontend/onnx/test_forward.py

```diff
@@ -95,23 +95,6 @@ def verify_onnx_forward_impl(graph_file, data_shape, out_shape):
         tvm.testing.assert_allclose(c2_out, tvm_out, rtol=1e-5, atol=1e-5)
 
 
-def verify_super_resolution_example():
```

Comment:
These tests are not used and references non existent variables like super_resolution, so I removed them.